banyand/observability/metrics_system.go

```diff
@@ -171,8 +171,7 @@ func collectDisk() {
 	for path := range getPath() {
 		usage, err := disk.Usage(path)
 		if err != nil {
-			// skip logging the case where the data has not been created
-			if err.Error() == "no such file or directory" {
+			if err.Error() != "no such file or directory" {
```

Comment:
No, checking the error message is not reliable. 
```suggestion
if _, err := os.Stat(path); err != nil {
			if !os.IsNotExist(err) {
				log.Error().Err(err).Msgf("failed to get stat for path: %s", path)
			}
			return
		}
```